types/es-abstract/2015/GetPrototypeFromConstructor.d.ts

```diff
@@ -0,0 +1,11 @@
+import type { Intrinsics } from '../GetIntrinsic';
+
+declare function GetPrototypeFromConstructor<K extends keyof Intrinsics>(
+    constructor: new (...args: any) => any,
+    intrinsicDefaultProto: K,
+): {} | Intrinsics[K];
+declare function GetPrototypeFromConstructor(
+    constructor: new (...args: any) => any,
+    intrinsicDefaultProto: string,
+): any;
```

Comment:
```suggestion
): null | object;
```
(these are the only valid prototype values)

Reply:
This is marked as resolved but still isn’t updated or explained.

Reply:
Because I’ve already fixed it in <https://github.com/DefinitelyTyped/DefinitelyTyped/pull/44805/commits/a366c979a63c8f4dd312bcf18d51d26903424587> 3 days ago.

Reply:
Did you forget to push that commit? If i click on it, it says "This commit does not belong to any branch on this repository.".

Reply:
nvm, perhaps it was rebased out, but at the time i left this comment it didn't seem to be included.